src/Agent.cs

```diff
@@ -25,6 +25,11 @@ namespace Instrumental
   public class Agent
   {
     /// <summary>
+    ///   The agent version, according to the agent.  See script/check_version for things that must match this
```

Comment:
The `script/check_version` bit seems like release documentation and not something we want to ship to customers for viewing in IDEs.